dask_jobqueue/slurm.py

```diff
@@ -134,7 +146,7 @@ def slurm_format_bytes_ceil(n):
         return "%dM" % math.ceil(n / (1024**2))
     if n >= 1024:
         return "%dK" % math.ceil(n / 1024)
-    return "1K" % n
+    return "%sK" % n
```

Comment:
I think the intent here is that is you enter a number like `34` for 34 bytes then it will automatically round up to `1K` which is the minimum that SLURM supports. So I think the actual fix is to just return `1K`.

```suggestion
    return "1K"
```